packages/components/addon/components/hds/text/index.js

```diff
@@ -0,0 +1,143 @@
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+
+export const AVAILABLE_ALIGNS = ['left', 'center', 'right'];
+
+export const AVAILABLE_COLORS = [
+  'primary',
+  'strong',
+  'faint',
+  'disabled',
```

Comment:
How do you think to approach indicating that the element is disabled? Because just text alone will be problematic for AT. The element would require the disabled attribute as well. Do we have documentation for the users on this at least?

Reply:
To me, this is just a descriptive word to represent the displayed color as a text component cannot itself be disabled.